src/regex/Regex.java

```diff
@@ -1,41 +1,66 @@
+    {
 
         // 1. What does the following pattern match? (\d){36} explain in a println() statement.
 
-        System.out.println("The following pattern ('\\d'){36} matches : ...");
+        System.out.println("The following pattern ('\\d'){36} matches : any string of 36 digits");
 
 
 
         String[] TEKmentors = {"Amir Yunas","Mark Bennet", "Rosa Garcia", "Desaree Byers", "Abram Jablonski", "Dylan Fellows", "Emilios Papas", "Jonathan Diamond"};
 
         // 2. Create a new array of the first names of the TEKmentors.  Use Regex to only grab the first name of every TEKmentor.  Push the values to a new array
         String[] tekMentors;
+        for (String firstWord: TEKmentors)
+        {
+            Pattern pattern = Pattern.compile("\\w+");
+            Matcher matcher = pattern.matcher(firstWord);
+            if (matcher.find())
+            {
+                firstWord = matcher.group();
+            }
+            System.out.println(firstWord);
+        }
 
 
         // 3. Find all the occurences of any form of 'book' in the following paragraph. use regex to match the occurences and store the count of books in an int.
         
-        int bookCount = 0;
+        long bookCount = 0;
         String bookText = "Books are the keys to knowledge.  I didn't like to read books as a child, but in college I started enjoying learning and reading books. You can borrows books from the library, or you can buy them from the bookstore. I'm not sure if I prefer paperback books or hardcover books.  I'm such a nerd that I even like textbooks.  With the advent of technology, you can even buy digital books, such as kindle-books, nook-books, or other e-books. My personal favorite book format are pdf-books, because I don't have to carry so many books around wherever I go.  All the books are on my ipad or laptop.  When I lived abroad, they would give books to students absolutely free.  Free books for a student of knowledge is like a kid in a candy store.  So wipe the dust off of your books, and remember the slogan from 'reading rainbow' : 'Take a look! It's in a book! Reading Rainbow!";
-        System.out.println(bookText);
+            Pattern pattern2 = Pattern.compile(("Book|book"));
+            Matcher matcher2 = pattern2.matcher(bookText);
+            bookCount = matcher2.results().count();
+            System.out.println("There are " + bookCount + " occurrences of 'book or Book' in the bookText string");
+//        System.out.println(bookText);
 
         // 4.a Create an array of all the words besides the word 'sleepy'.  Each word does not have to be a separate element, although you can split it that way if you wish.  We just want an array that everything that is not 'sleepy'.  
 
         String sleepy = "I felt sleepy because I saw the others were sleepy and because I knew I should feel sleepy, but I wasn't really sleepy.  If you're sleepy and you know it, clap your hands.  Keep on being sleepy until you actually become sleepy";
```

Comment:
Though your solution is much cleaner, the expectation was a String[] of the words in this sentence.